src/components/range/range.ts

```diff
@@ -360,6 +365,9 @@ export class Range extends Ion implements AfterViewInit, ControlValueAccessor, O
       return false;
     }
 
+    // trigger ionFocus event
+    this.ionFocus.emit(this);
```

Comment:
I think it would be more correct there, as if I put this at the end, the order of firing would be:
`ionChange` -> `ionFocus` -> `ionChange` x many -> `ionBlur`

Instead of:
`ionFocus` -> `ionChange` x many -> `ionBlur`

Still think I should change?